templates/fpm/php-fpm.conf.erb

```diff
@@ -37,12 +37,18 @@ error_log = /var/log/php5-fpm.log
 ; See syslog(3) for possible values (ex daemon equiv LOG_DAEMON)
 ; Default Value: daemon
 ;syslog.facility = daemon
+<% if @syslog_facility %>
+syslog.facility = <%= @syslog_facility %>
+<% end %>
```

Comment:
okay, so, this is a matter of personal taste, and you can ignore it if you disagree, but I'd rather have:
```erb
<%- if @syslog_facility -%>
syslog.facility = <%= @syslog_facility %>
<%- else -%>
; syslog.facility = daemon
<%- end -%>
```
otherwise the end result makes for a confusing read else, imo.